### ISSUE TITLE
How to change line spacing

### ISSUE THREAD

**Issue by barelief** [ifrost/afterwriting-labs#125]
Hello! 
just wondering if there is a way to change the line spacing (single, 1.5, double, etc) in PDF output?

**Comment by ifrost** [comment]
Hi! No, it's not possible as it'd give an output not conforming to the industry standards. Could you give an example when it'd be useful?

**Comment by barelief** [comment]
i found it useful to print with bigger line space when giving script to the
editor, so he can mark his revision notes.

**Comment by ifrost** [comment]
Good point! I think it'd be actually quite useful. Do you think it'd be better to change line spacing and at the same time make the font smaller so the final PDF results in the same number of pages?

**Comment by barelief** [comment]
Nope. It's better to have more pages for the sake of better readability.
Any editor would love to have a good font size and proper line spacing.

Thanks!
